### PR TITLE
Initial upload of new site

### PR DESCRIPTION
**Pull request by insectengine**
Let's give this a whirl!

### CODE REVIEW

**Comment by lholmquist** [comment]
Cool!!  i'll take a look

**Comment by lholmquist** [comment]
in the section that talks about publishing to GH pages, https://github.com/insectengine/nodeshift.github.io/tree/new-site#deployment-on-github-pages,  do we need to do that?

Also,  the CNAME is being removed in this PR,  do we need that?

File: _data/authors.yaml [hunk 1]
```diff
@@ -0,0 +1,14 @@
+tsmith:
+  name: "Tim Smith"
+  email: "tsmith@email.com"
+  emailhash: "55acsfsfg7f29105b0"
+  job_title: ""
+  twitter: "tsmith"
+  bio: "Lorem ipsum dolor sit amet, eos posse ipsum legendos id. Ad duo assum augue democritum."
```
Comment (lance) [inline _data/authors.yaml:7]: Assuming these bios are not part of the published site.

File: README.md [hunk 2]
```diff
@@ -0,0 +1,67 @@
...
+1. The Jekyll site provides detailed installation instructions for each operating system:
+ 
+  - [Mac](https://jekyllrb.com/docs/installation/macos/)
+  - [Linux distributions including Red Hat Linux](https://jekyllrb.com/docs/installation/other-linux)
+  - [Ubuntu Linux](https://jekyllrb.com/docs/installation/ubuntu/)
+  - [Windows](https://jekyllrb.com/docs/installation/windows/)
+    
+3. Fork this repository by clicking the _Fork_ button at the top right corner of this page.
+4. Clone your fork (please ensure you have current version of git installed) by running: 
+  `git clone git@github.com:YOUR_USER_NAME/community-theme.git`
```
Comment (lance) [inline README.md:28]:
```suggestion
  `git clone git@github.com:YOUR_USER_NAME/nodeshift.github.io.git`
```

File: README.md [hunk 3]
```diff
@@ -0,0 +1,67 @@
...
+5. Change into the project directory
+  `cd community-theme`
```
Comment (lance) [inline README.md:30]:
```suggestion
  `cd nodeshift.github.io`
```

**Comment by insectengine** [comment]
I spoke with Adela about this and she said with this site, we shouldn't need the GithubPages Gem. The CNAME should be included.

File: README.md [hunk 4]
```diff
@@ -0,0 +1,67 @@
...
+  `bundle exec jekyll serve`
+7. To preview your site, browse to http://localhost:4000
+
+> If you encounter any unexpected errors during the above, please refer to the [troubleshooting](https://jekyllrb.com/docs/troubleshooting/#configuration-problems) page or the [requirements](https://jekyllrb.com/docs/installation/#requirements) page, as you might be missing development headers or other prerequisites.
+
+_For more information regarding the use of Jekyll, please refer to the [Jekyll Step by Step Tutorial](https://jekyllrb.com/docs/step-by-step/01-setup/)._
+
+## Deployment on GitHub Pages
+
+To deploy your site using GitHub Pages you will need to add the [github-pages gem](https://github.com/github/pages-gem).
```
Comment (lance) [inline README.md:41]: I could easily be mistaken, but I believe there is quite a bit more involved with actually deploying. Currently the nodeshift.dev site is published on the `master` branch at `/`. If I understand correctly, to deploy this generated site, doesn't the generated output need to either be on the `github-pages` branch or on any other branch (including `master`) in either `/` or `/docs`? 

Or perhaps I'm missing something. Maybe using Jekyll in combination with GitHub that is magic that just happens. :shrug: 

**EDIT**: Looking at the GitHub pages documentation, it seems like they handle this magic - so this comment is not relevant, probably.

File: _config.yml [hunk 5]
```diff
@@ -0,0 +1,162 @@
...
+# in the templates via {{ site.myvariable }}.
+title: NodeShift
+email:
+description: >- # this means to ignore newlines until "baseurl:"
+  GET YOUR NODE.JS ON OPENSHIFT - Nodeshift is an opinionated command line application and programmable API that you can use to deploy Node.js projects to OpenShift.
+# baseurl: "/nodeshift_website" # the subpath of your site, e.g. /blog
+url: "https://nodeshift.dev/" # the base hostname & protocol for your site, e.g. http://example.com
+twitter_username: 
+github_username:  nodeshift
+github_fork_url: "https://github.com"
```
Comment (lance) [inline _config.yml:24]: Shouldn't this be?
```suggestion
github_fork_url: "https://github.com/nodeshift/nodeshift.github.io"
```

**Review by lance — COMMENTED** [review]
Should probably remove .DS_Store files and add that to .gitignore.

I think `CNAME` should not be removed, as that's what allows us to have the site available at https://nodeshift.dev.

Other than that and a few comments inline, it looks great!

File: _data/authors.yaml (hunk 1 above)
Comment (lholmquist) [reply _data/authors.yaml:7]: Yea, we will need to update these stock things.

I told james lets do the handoff and my team can update the rest of the small bits

**Comment by lholmquist** [comment]
I'm going to merge this in, since it is just going to the non-master branch.   Then from there we can address any outstanding issues

**Pull request merged**